scipy/signal/signaltools.py

```diff
@@ -94,6 +94,13 @@ def _inputs_swap_needed(mode, shape1, shape2):
 
     return False
 
+def _reshape_nd(x1d, ndim, axis):
+    """
+    Reshape x1d to size 1 along all axes in ``range(ndim)`` except for ``axis``.
+    """
+    shape = [1] * ndim
+    shape[axis] = x1d.size
+    return x1d.reshape(shape)
```

Comment:
PEP8, two blank lines after function def
```suggestion
    return x1d.reshape(shape)

```